hack/update_kep_overview.sh

```diff
@@ -31,7 +31,7 @@ do
 
   # Extract fields from header
   KEP_NUMBER=$(echo "$KEP_HEADER" | sed -n -E 's/kep-number: ([0-9]+)/\1/p')
-  KEP_NUMBER=$(echo "$KEP_NUMBER" | sed 's/^0*//')  # Strip leading zeros, or we interpret in octal
+  KEP_NUMBER="$((10#$KEP_NUMBER))"    # Strip leading zeros, or we interpret in octal
```

Comment:
Is there a way we can keep this as it was? The `echo` and `sed` makes it pretty clear what's happening, but this new way seems kind of exotic to me?

Not a blocker though, the comment says what it does, so i guess it's ok

Reply:
@ANeumann82 I actually think what you had was easier to read or understand as well :(  shellcheck failure